### PR TITLE
Modified add goals page to look like the design

### PR DESCRIPTION
**Pull request by GithinjiHans**

### CODE REVIEW

**Comment by pulkitvyas08** [comment]
@GithinjiHans The add goals page doesn't look like the one from #213. Attaching the preview version for reference

image

File: src/components/todoList/TodoListForm.tsx [hunk 1]
```diff
@@ -53,16 +56,16 @@ export function TodoForm() {
...
       <div>
         <input
           className={darkModeStatus ? 'addtaskdark-time' : 'addtasklight-time'}
-          type="date"
-          placeholder="Add time"
+          type="text"
+          placeholder={t('addTimePlaceholder')}
           name="inputTime"
           value={formInputData.inputTime}
           onChange={handleChange}
```
Comment (pulkitvyas08) [inline src/components/todoList/TodoListForm.tsx:71]: Refer to #213 for the UI snippet

**Review by pulkitvyas08 — CHANGES_REQUESTED** [review]

File: src/components/todoList/TodoListForm.tsx (hunk 1 above)
Comment (GithinjiHans) [reply src/components/todoList/TodoListForm.tsx:71]: I didn't realise that there was a brand new design....I'll close this PR and work on the design tomorrow.

**Comment by tijlleenders** [comment]
No need to close the pull request - just add the required changes.

**Review by pulkitvyas08 — CHANGES_REQUESTED** [review]
Why is there a "change color" button which does exactly that? This is not part of the design

**Comment by tijlleenders** [comment]
@pulkitvyas08 There was an update to the design #213 so the 'Change colour' button is now part of the design.  
image

But I agree it still doesn't match exactly:
image

Still todo in my opinion:
- The subtext "Add your task in format ..."
- Change the colours to match the ZinZen colour theme

OK not to implement in this PR:
-  You can leave out the missing the suggestion buttons for duration and repeat. They are dynamic depending on the text typed and depend on #273

**Comment by tijlleenders** [comment]
Resolves #308

**Comment by tijlleenders** [comment]
Resolves #295

**Comment by pulkitvyas08** [comment]
> 

Oh Okay I did not check the new design, my bad. So @GithinjiHans please make the required changes and mark the PR `Ready for review` to get it reviewed

**Comment by GithinjiHans** [comment]
@pulkitvyas08 I have made the changes and I think that it's ready for review now

**Comment by pulkitvyas08** [comment]
@GithinjiHans @tijlleenders I'm really confused about what this code is supposed to do. There are no models or interfaces or API for this, the created tasks are not being added to any collection in the database, there is no validation for the `task + duration + frequency` format and obviously, there is no persistence here. 

So I just wanted to know the objective here, is this only to update the design? If yes then please point me to the relevant issues and PRs that are solving the above problems.

File: src/components/todoList/TodoListForm.tsx [hunk 2]
```diff
@@ -37,7 +42,7 @@ export function TodoForm() {
     }
```
Comment (pulkitvyas08) [inline src/components/todoList/TodoListForm.tsx:42]: This is not saving anywhere

**Review by pulkitvyas08 — CHANGES_REQUESTED** [review]

File: src/components/todoList/TodoListForm.tsx (hunk 2 above)
Comment (GithinjiHans) [reply src/components/todoList/TodoListForm.tsx:42]: Yea...we were just doing the design first then we'll handle persistence later

**Comment by tijlleenders** [comment]
> @GithinjiHans @tijlleenders I'm really confused about what this code is supposed to do. There are no models or interfaces or API for this, the created tasks are not being added to any collection in the database, there is no validation for the `task + duration + frequency` format and obviously, there is no persistence here.
> 
> So I just wanted to know the objective here, is this only to update the design? If yes then please point me to the relevant issues and PRs that are solving the above problems.

This PR solves #295, as stated in the beginning of the pull request.
Looking at that issue:
image

So yes, this is only about updating the design.
Persistence and deleting is explicitly mentioned as not being part of the work.
This is because the API was still being developed #294 but also because we're building up in small increments. 

For example, adding validation can be a separate issue. It can even be done by someone else. Small steps are necessary so that issues can actually be finished on the day they are started. Skills vary, and I think this fits better with the availability of everyone. It is rare that someone can work on the project for a whole day.

I think the real issue is that we need a better view of the total product.
Mallika will give a demo of the vision tomorrow. Hope you can make it.
If we know how the pieces fit together, we can also make them one by one.

**Comment by tijlleenders** [comment]
@GithinjiHans I think it it's nearly there.
Nice to see you also included the internationalization.

If you clean this up it looks good from the UI perspective for me.
image

**Comment by pulkitvyas08** [comment]
> This PR solves #295, as stated in the beginning of the pull request. Looking at that issue: image
> 
> So yes, this is only about updating the design. Persistence and deleting is explicitly mentioned as not being part of the work. This is because the API was still being developed #294 but also because we're building up in small increments.

Okay, I think I get the issue now. The problem here is the terminology being used in #295, this PR, and the codebase. In the code, the files are named as `TodoList.tsx` with variables and class names like `add-task`. This naming inconsistency made me think that goals and tasks are two different things

**Comment by GithinjiHans** [comment]
I think we're good to go now...

**Review by pulkitvyas08 — APPROVED** [review]
Yes, let's take up the rest of the problems in the next PR. Great start to the feature @GithinjiHans!

**Pull request merged**